.github/workflows/tests.yml

```diff
@@ -47,7 +48,7 @@ jobs:
         shell: bash
         run: |
           build_scripts/server_start.sh
-          pytest --verbose -m "not integration and not smoke" --timer-top-n 10 -n auto --cov=neuro_san
+          pytest --verbose -m "not integration and not smoke" --timer-top-n 10 -n auto --cov-config=.coveragerc --cov
```

Comment:
Use a .coveragerc instead of piling the coverage needs on the pytest command line
This makes tests.yml more repo-agnostic.

In the end, the only thing that makes this repo's tests.yml special is the call to ./build_scripts/server_start.sh